### ISSUE TITLE
Fix image layout = "full-width"and a few more nits in Design Bitcoin Applications

### ISSUE THREAD

**Issue by pavlenex** [BitcoinDesign/Guide#370]
@mau211 thanks for the contribution. Quick heads up that the images are not the right sizes, the alt text should be updated to describe the image content, and `layout = "full-width"` is added 4 times on the include (once is enough). I can create a PR tomorrow to fix things up.

_Originally posted by @GBKS in https://github.com/BitcoinDesign/Guide/issues/336#issuecomment-854047698_
